README.md

```diff
@@ -27,8 +27,9 @@ In order to run GraphSpace, please install sqlite3 and both the Python runtime a
 3. Create a virtual environment for the project: `virtualenv venv`
 4. Start using the virtual environment: `source venv/bin/activate`
 5. Install graphspace: `sh install.sh`
-6. Finally, start the GraphSpace server: `python manage.py runserver --settings=graphspace.settings.local`
-7. Visit `http://localhost:8080` and enjoy using GraphSpace!
+6. Change the postgres user credentials in DATABASES in `/graphspace/settings/local.py` 
```

Comment:
I meant can describe how the user can change these credentials ? I mean the give some sample parameter value and provide the name of these parameters